src/sat.cc

```diff
@@ -541,6 +541,14 @@ bool Sat::PrintVersion() {
 // This needs to be called before Run(), and after ParseArgs().
 // Returns true on success, false on error, and will exit() on help message.
 bool Sat::Initialize() {
+  test_run_ = std::make_unique<ocpdiag::results::TestRun>(
```

Comment:
Is this a comment on the design of the results v2 lib? I think it's better practice to pass an object rather than a long list of arguments, but either way I don't think it's relevant to this PR.